packages/mobile/src/screens/chat-screen/ChatBlastFollowersCTA.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useCallback } from 'react'
+
+import { useCanSendChatBlast } from '@audius/common/hooks'
+import { chatActions } from '@audius/common/store'
+import { ChatBlastAudience } from '@audius/sdk'
+import { TouchableHighlight } from 'react-native-gesture-handler'
+import { useDispatch } from 'react-redux'
+
+import {
+  Box,
+  Flex,
+  Text,
+  IconTowerBroadcast,
+  IconCaretRight
+} from '@audius/harmony-native'
+
+const { createChatBlast } = chatActions
+
+const getMessages = (audience: ChatBlastAudience) => {
+  switch (audience) {
+    case ChatBlastAudience.FOLLOWERS:
+      return {
+        title: 'Message Blast Your Followers',
+        description: 'Send bulk messages to your followers.'
+      }
+    case ChatBlastAudience.TIPPERS:
+      return {
+        title: 'Message Blast Your Tip Supporters',
+        description: 'Bulk message everyone who’s sent you a tip.'
+      }
+  }
+}
+
+type ChatBlastWithAudienceCTAProps = {
+  audience: ChatBlastAudience
+}
+
+export const ChatBlastWithAudienceCTA = (
+  props: ChatBlastWithAudienceCTAProps
```

Comment:
nit - i would destructure props inline for just 1 prop

Reply:
I could have sworn it's in our style guide to do it in the body, but I'm personally fine with either

Reply:
idc lol just personal pref